nfs-ganesha/build/build_rpm

```diff
@@ -92,7 +92,7 @@ echo "\"\"\"" >> nfs-ganesha.cfg
 echo "Building RPMs"
 sudo mock --verbose -r nfs-ganesha.cfg --define "dist .el7" --resultdir=$WORKSPACE/dist/RPMS/ ${SRPM} || ( tail -n +1 $WORKSPACE/dist/RPMS/{root,build}.log && exit 1 )
 
-VERSION=`grep -om 1 '[[:digit:]]\+\.[[:digit:]]\+-dev-[[:digit:]]\+' $WORKSPACE/nfs-ganesha/build/include/config.h`
+VERSION=`grep -R "#define GANESHA_VERSION \"" $WORKSPACE/nfs-ganesha/build/include config.h | sed -e 's/#define GANESHA_VERSION "//1; s/"//1;'`
```

Comment:
is the path here part of the problem this is fixing? e.g. `nfs-ganesha/build/include` and `config.h` or is there a typo here with a missing `/` so that the path is `nfs-ganesha/build/include/config.g` ?